uninstall.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Fired when the plugin is uninstalled.
+ *
+ * @link       https://nataliesmith.dev
+ * @since      1.0.0
+ * @package    Poetwp
+ */
+
+// If uninstall not called from WordPress, then exit.
+if ( ! defined( 'WP_UNINSTALL_PLUGIN' ) ) {
+	exit;
+}
+
+/**
+ * Deletes all poems from the database when the plugin is uninstalled.
+ */
+$args = array(
+	'post_type'      => 'poem',
+	'posts_per_page' => -1,
```

Comment:
Aside: This will not scale on a large site with many poems if you were to try and delete all of them.